shop/src/main/java/ro/msg/learning/shop/entity/Order.java

```diff
@@ -0,0 +1,30 @@
+package ro.msg.learning.shop.entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
+@AllArgsConstructor
+public class Order extends BaseEntity{
+
+    @ManyToOne
+    @JoinColumn(name = "ShippedFromId")
+    private Location shippedFrom;
+
+    @ManyToOne
+    @JoinColumn(name = "CustomerId")
+    private Customer customer;
+
+    private LocalDateTime createdAt;
+    private String addressCountry;
+    private String addressCounty;
+    private String addressCity;
+    private String addressStreetAddress;
```

Comment:
Use here the new entity "Address" mentioned before.

Reply:
Fixed